services/ui-src/src/components/layout/Header.tsx

```diff
@@ -20,7 +20,12 @@ export const Header = ({ handleLogout }: Props) => {
               <Image src={appLogo} alt="MCR logo" sx={sx.appLogo} />
             </Link>
             <Flex sx={sx.menuFlex}>
-              <Link as={RouterLink} to="/help" data-testid="header-help-button">
+              <Link
+                as={RouterLink}
+                to="/help"
+                aria-label="get help"
```

Comment:
Added `aria-label` to fix accessibility issue at mobile size ("Get Help" text is only shown on desktop).